docs/self-managed/components/orchestration-cluster/core-settings/configuration/properties.md

```diff
@@ -12,135 +12,162 @@ As a Spring Boot application, the Orchestration Cluster supports standard
-| `CAMUNDA_SECURITY_MULTITENANCY_APIENABLED`                      | Enables the multi-tenancy API and UI independently from multi-tenancy checks.  | `true`        |
+### `camunda.security.authentication`
+
+| Property                                                          | Description                                                                                                                                                                                                                                                                                                              | Default value |
+| ----------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------------- |
+| `camunda.security.authentication.authentication-refresh-interval` | The interval at which the memberships (groups, roles, tenants, component authorizations) are refreshed for logged in users. Find more details in [webserver and security](/self-managed/components/orchestration-cluster/core-settings/configuration/webserver.md#propagation-of-membership-changes-to-active-sessions). | `PT30S`       |
+| `camunda.security.authorizations.enabled`                         | If authorizations are enabled.                                                                                                                                                                                                                                                                                           | `true`        |
+| `camunda.security.authentication.method`                          | The authentication method to use. Options: `basic`, `oidc`.                                                                                                                                                                                                                                                              | `basic`       |
+| `camunda.security.authentication.unprotected-api`                 | If the API can be used without authentication.                                                                                                                                                                                                                                                                           | `false`       |
+
+### `camunda.security.csrf`
+
+| Property                        | Description                          | Default value |
+| ------------------------------- | ------------------------------------ | ------------- |
+| `camunda.security.csrf.enabled` | Enables or disables CSRF protection. | `true`        |
+
+#### Default CSRF Protection
```

Comment:
This is not a real property. I would tend to move this to a dedicated section and create a link from the property description